roles/vault/README.md

```diff
@@ -40,6 +41,18 @@ Example playbook (used with OpenStack Kayobe)
   gather_facts: True
   hosts: consul
   tasks:
+    - name: Copy rootCA
+      copy:
+        content: "{{ external_ca_cert }}"
+        dest: "{{ '/etc/pki/ca-trust/source/anchors/rootCA.crt' if ansible_os_family == 'RedHat' else '/usr/local/share/ca-certificates/rootCA.crt' }}"
+        mode: 0600
+        no_log: True
+      become: true
+
+    - name: update system CA
+      become: true
+      shell: "{{ 'update-ca-trust' if ansible_os_family == 'RedHat' else 'update-ca-certificates' }}"
```

Comment:
Does it need to be in the system trust store?

Reply:
Yes, the URI module in vault.yml can't talk to vault if you use self-signed certificates.